examples/sshguard.conf.sample

```diff
@@ -9,12 +9,17 @@
 #BACKEND="/usr/local/libexec/sshg-fw-iptables"
 
 # Space-separated list of log files to monitor. (optional, no default)
-#FILES="/var/log/auth.log /var/log/authlog /var/log/maillog"
+# debian-style systems
+#FILES="/var/log/mail.log /var/log/auth.log"
+# slackware and redhat-style systems
+#FILES="/var/log/maillog /var/log/secure"
```

Comment:
This is highly OS-dependent and can change over time. I don't think it's necessary to give such examples for every OS.